lib/facter/consul.rb

```diff
@@ -0,0 +1,24 @@
+require 'facter'
+
```

Comment:
Seems like this should be "confined" on something. I guess not really because it could be windows/mac/linux